cmd/openshift-dev-helpers/main.go

```diff
@@ -35,7 +35,7 @@ func NewCmdDevHelpers(streams genericclioptions.IOStreams) *cobra.Command {
 		Use:          "openshift-dev-helpers",
 		Short:        "Set of helpers for OpenShift developer teams",
 		SilenceUsage: true,
-		RunE: func(c *cobra.Command, args []string) error {
+		PreRunE: func(c *cobra.Command, args []string) error {
```

Comment:
That's not a good fix, `Run` is meant for executing the core functionality of a command, and that's what we're currently doing here.

Reply:
cool updated @soltysh